Core/StartupArgManager.cs

```diff
@@ -0,0 +1,40 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace LocalAdmin.V2.Core
+{
+    public sealed class StartupArgManager
+    {
+        /// <summary>
+        /// Path to startup arguments file.
+        /// </summary>
+        private const string StartupArgsPath = "laargs.txt";
+
+        /// <summary>
+        /// Merges Command-line arguments and arguments in <paramref name="StartupArgsPath"/>
+        /// </summary>
+        /// <param name="CMDArgs">Runtime Command-Line Arguments</param>
+        /// <returns>Merged Arguments</returns>
+        public static string[] MergeStartupArgs(string[] CMDArgs)
+        {
+            List<string> StartupArgs = new List<string>();
+            foreach (string arg in CMDArgs)
+            {
+                if (!StartupArgs.Contains(arg))
+                    StartupArgs.Add(arg);
```

Comment:
AddRange can be used instead of this entire foreach loop.

Reply:
Done, and changed for CMDArgs.